convert.sh

```
@@ -0,0 +1,42 @@
+#!/bin/bash
+
+# convert.sh
+#
+# このスクリプトは convert.py を実行するためのシェルスクリプトです。
+# 必要に応じて仮想環境を有効化し、Pythonスクリプトを実行します。
+
+# スクリプトが存在するディレクトリに移動
+cd "$(dirname "$0")"
```

Shellcheck (0.10.0)

[warning] Use 'cd ... || exit' or 'cd ... || return' in case cd fails.

(SC2164)